src/dynarec/rv64/dynarec_rv64_00_3.c

```diff
@@ -910,6 +931,7 @@ uintptr_t dynarec64_00_3(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
             switch(tmp) {
                 case 3:
                     SETFLAGS(X_ALL, SF_SET_NODF);    // Hack to set flags to "dont'care" state
+                    NAT_FLAGS_NOFUSION();
```

Comment:
Oof, need to add a command to allow or not native flags fusion? Why not change `SETFLAGS` instead?